integration/emr/alluxio-emr.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#Create user and download release
+sudo groupadd alluxio -g 600
+sudo useradd alluxio -u 600 -g 600
+if [ -z $1]
+then
+  wget http://downloads.alluxio.io/downloads/files/2.0.0-preview/alluxio-2.0.0-preview-bin.tar.gz
+else
+  wget "${1}"
+fi
+sudo cp alluxio-*.tar.gz /opt/
```

Comment:
I would avoid using `alluxio-*.tar.gz` here, in case there are other alluxio tarball in the same directory by chance. Let's try to be explicit on the file name.

Reply:
Fixed.